internal/server/env_vars.go

```diff
@@ -31,15 +32,16 @@ func NewEnvironmentVars() (EnvironmentVars, error) {
 	}
 
 	return EnvironmentVars{
-		Port:            getEnv("PORT", "1234"),
-		WebDir:          getEnv("WEB_DIR", "web"),
-		SiriusURL:       getEnv("SIRIUS_URL", "http://localhost:8080"),
-		SiriusPublicURL: getEnv("SIRIUS_PUBLIC_URL", "http://localhost:8080"),
-		FirmHubURL:      getEnv("FIRM_HUB_HOST", "") + "/supervision/deputies/firm",
-		Prefix:          getEnv("PREFIX", ""),
-		DefaultPaTeam:   defaultPaTeamId,
-		DefaultProTeam:  defaultProTeamId,
-		Features:        strings.Split(getEnv("FEATURES", ""), ","),
+		Port:             getEnv("PORT", "1234"),
+		WebDir:           getEnv("WEB_DIR", "web"),
+		SiriusURL:        getEnv("SIRIUS_URL", "http://localhost:8080"),
+		SiriusPublicURL:  getEnv("SIRIUS_PUBLIC_URL", "http://localhost:8080"),
+		FirmHubURL:       getEnv("FIRM_HUB_HOST", "") + "/supervision/deputies/firm",
+		Prefix:           getEnv("PREFIX", ""),
+		DefaultPaTeam:    defaultPaTeamId,
+		DefaultProTeam:   defaultProTeamId,
+		Features:         strings.Split(getEnv("FEATURES", ""), ","),
+		FinanceAdminLink: getEnv("FINANCE_ADMIN_LINK", "1"),
```

Comment:
I think it would be better to default this to toggled off, otherwise if the flag is not set to off in prod for some reason, it will be defaulted to on.